prime-router/src/main/kotlin/tokens/OktaAuthentication.kt

```diff
@@ -2,53 +2,122 @@ package gov.cdc.prime.router.tokens
 
 import com.google.common.net.HttpHeaders
 import com.microsoft.azure.functions.ExecutionContext
+import com.microsoft.azure.functions.HttpMethod
 import com.microsoft.azure.functions.HttpRequestMessage
 import com.microsoft.azure.functions.HttpResponseMessage
+import com.okta.jwt.JwtVerificationException
+import com.okta.jwt.JwtVerifiers
 import gov.cdc.prime.router.Organization
 import gov.cdc.prime.router.azure.ActionHistory
-import gov.cdc.prime.router.azure.AuthenticatedClaims
-import gov.cdc.prime.router.azure.AuthenticationVerifier
 import gov.cdc.prime.router.azure.HttpUtilities
-import gov.cdc.prime.router.azure.OktaAuthenticationVerifier
-import gov.cdc.prime.router.azure.PrincipalLevel
-import gov.cdc.prime.router.azure.TestAuthenticationVerifier
 import gov.cdc.prime.router.azure.WorkflowEngine
 import gov.cdc.prime.router.common.Environment
 import org.apache.logging.log4j.kotlin.Logging
 
+// These constants match how PRIME Okta subscription is configured
+const val oktaGroupPrefix = "DH"
+const val oktaSenderGroupPrefix = "DHSender_"
+const val oktaAdminGroupSuffix = "Admins"
+const val oktaSystemAdminGroup = "DHPrimeAdmins"
```

Comment:
There are multiple definitions of this string in the codebase.  Consolidate in the best place.

Reply:
done.   There are more references in HistoryFunction.kt that will go away when we replace that code.